### PR TITLE
Remove Tutorial and Add Saving Data manual test

### PR DESCRIPTION
**Pull request by DennieDan**

### CODE REVIEW

File: docs/SettingUp.md [hunk 1]
```diff
@@ -52,11 +52,4 @@ If you plan to use Intellij IDEA (highly recommended):
...
 
-   When you are ready to start coding, we recommend that you get some sense of the overall design by reading about [AddressBook’s architecture](DeveloperGuide.md#architecture).
-
-1. **Do the tutorials**
-   These tutorials will help you get acquainted with the codebase.
-
-   * [Tracing code](tutorials/TracingCode.md)
-   * [Adding a new command](tutorials/AddRemark.md)
-   * [Removing fields](tutorials/RemovingFields.md)
+   When you are ready to start coding, we recommend that you get some sense of the overall design by reading about [UGTeach’s architecture](DeveloperGuide.md#architecture).
```
Comment (Feng1231) [inline docs/SettingUp.md:55]: I think I have already updated this in #280 😓

**Review by Feng1231 — COMMENTED** [review]
I think I have removed the links in one of my PR

**Comment by Feng1231** [comment]
> Sorry to say but I think Chen Feng has already done all that this PR does in #280

I think for manual testing, the more the merrier

**Review by teamotitan — COMMENTED** [review]
lgtm, and there is Mr CF message

**Pull request closed without merging**